wpilibc/src/main/native/cpp/estimator/DifferentialDrivePoseEstimator.cpp

```diff
@@ -0,0 +1,109 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/estimator/DifferentialDrivePoseEstimator.h"
+
+#include <array>
```

Comment:
This include should be in the header instead because there's a function prototype that uses std::array.